src/EditorFeatures/Core/InlineRename/InlineRenameService.cs

```diff
@@ -85,6 +86,12 @@ internal sealed class InlineRenameService(
             return new InlineRenameSessionInfo(renameInfo.LocalizedErrorMessage);
         }
 
+        var symbolService = document.GetRequiredLanguageService<IGoToDefinitionSymbolService>();
+        var (symbol, _, _) = await symbolService.GetSymbolProjectAndBoundSpanAsync(
+            document, textSpan.Start, cancellationToken).ConfigureAwait(false);
```

Comment:
this is also problematic.  InlineRename is a language agnostic feature (can be used yb any roslyn language).  but symbols are a C#/VB concept.  This is pulling .Net'isms into core shared feature code.  this needs to be extracted out such that any language can participate here. (we use ILanguageServices for this).